src/studio/src/designer/backend/Services/Implementation/ConvertTextsService.cs

```diff
@@ -18,14 +18,16 @@ namespace Altinn.Studio.Designer.Services.Implementation
     public class ConvertTextsService : IConvertTextsService
     {
         private readonly IAltinnGitRepositoryFactory _altinnGitRepositoryFactory;
+        private readonly ITextsService _textsService;
 
         /// <summary>
         /// Constructor
         /// </summary>
         /// <param name="altinnGitRepositoryFactory">IAltinnGitRepository</param>
-        public ConvertTextsService(IAltinnGitRepositoryFactory altinnGitRepositoryFactory)
+        public ConvertTextsService(IAltinnGitRepositoryFactory altinnGitRepositoryFactory, ITextsService textsService)
```

Comment:
The fact that you need the ITextsService in the ConvertTextsService and that you had to change a private method to public to use it here fall back to the comment earlier that the convert logic should be in the ITextsService. They are both operating on the same entities.